docs/development/README.md

```diff
@@ -91,21 +91,21 @@ npm run test:contract
 
 Note: A test database is required to run contract tests. You can quickly set one up using `bin/setup_test_db.sh`
 
-## Running e2e tests
+## Running E2E tests
```

Comment:
I think it's good practice to say what this acronym means as it might not be know to someone new.

```suggestion
## Running end to end (E2E) tests
```

Reply:
Woops I missed that one, good eye